packages/react-i18n/README.md

```diff
@@ -0,0 +1,226 @@
+# `@shopify/react-i18n`
+
+[![Build Status](https://travis-ci.org/Shopify/quilt.svg?branch=master)](https://travis-ci.org/Shopify/quilt)
+[![License: MIT](https://img.shields.io/badge/License-MIT-green.svg)](LICENSE.md) [![npm version](https://badge.fury.io/js/%40shopify%2Freact-i18n.svg)](https://badge.fury.io/js/%40shopify%2Freact-i18n.svg) [![npm bundle size (minified + gzip)](https://img.shields.io/bundlephobia/minzip/@shopify/react-i18n.svg)](https://img.shields.io/bundlephobia/minzip/@shopify/react-i18n.svg)
+
+i18n utilities for React handling translations, formatting, and more.
```

Comment:
We might want something in the docs (FAQ?) that amounts to "Why another translation library?" since we get that question a lot it would be nice to be able to link to an answer.